tests/metrics_test.py

```diff
@@ -1583,6 +1585,7 @@ def test_calculate_global_stats(self):
         self.assertEqual(collections.OrderedDict(
             [("50_0", 200), ("100_0", 210), ("mean", 200), ("unit", "ms")]), opm["service_time"])
         self.assertAlmostEqual(0.3333333333333333, opm["error_rate"])
+        self.assertAlmostEqual(709*1000, opm["duration"])
```

Comment:
Shouldn't we able to use `assertEqual` here as this is an integer value?

Reply:
Opps thank you for catching